bucket/screen-saver-control.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.61",
+    "description": "A small utility that allows deactivate or activate both screen saver and monitor power off with a simple click.",
+    "homepage": "http://www.ampsoft.net/utilities/ScreenSaverControl.php",
+    "license": "Freeware",
+    "url": "http://www.ampsoft.net/files/ScreenSaverControl.zip",
+    "hash": "26ab07326d91db6a80fc9de77dedbc9b36bb3070c514a1be7239ea397beb512f",
+    "bin": "ScreenSaverControl.exe",
+    "shortcuts": [
+        [
+            "ScreenSaverControl.exe",
+            "Screen Saver Control"
+        ]
+    ],
+    "checkver": "Version ([\\d.]+)",
+    "autoupdate": {
+        "url": "http://www.ampsoft.net/files/ScreenSaverControl.zip"
+    }
```

Comment:
Latest version 2009

```suggestion
    ]
```

Reply:
Thanks!

Reply:
Stop resolving conversations, when you do not commit changes...

Reply:
Commenting on resolved conversations is nonsense as it does not get revealed, so it is completely waste notification, which noone would see...

Reply:
Thank you for your guidance :blush: